<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionDerivation</class>
 <widget class="QMainWindow" name="FunctionDerivation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FunctionDerivation</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonLB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRB">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMinus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFois">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Ã—</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonX1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEqual">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonX2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDivision">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRoot">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>âˆš</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>70</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>pow</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{
color: black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearAll">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="derivationDegreeLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Derivation degree</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="derivationDegreeSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <zorder>lineEdit</zorder>
   <zorder>pushButtonLB</zorder>
   <zorder>pushButtonRB</zorder>
   <zorder>pushButtonCos</zorder>
   <zorder>pushButtonSin</zorder>
   <zorder>pushButton7</zorder>
   <zorder>pushButton8</zorder>
   <zorder>pushButtonPlus</zorder>
   <zorder>pushButton9</zorder>
   <zorder>pushButton4</zorder>
   <zorder>pushButton5</zorder>
   <zorder>pushButtonMinus</zorder>
   <zorder>pushButton6</zorder>
   <zorder>pushButton1</zorder>
   <zorder>pushButton2</zorder>
   <zorder>pushButtonFois</zorder>
   <zorder>pushButton3</zorder>
   <zorder>pushButtonX1</zorder>
   <zorder>pushButton0</zorder>
   <zorder>pushButtonEqual</zorder>
   <zorder>pushButtonX2</zorder>
   <zorder>pushButtonDivision</zorder>
   <zorder>pushButtonRoot</zorder>
   <zorder>pushButtonExp</zorder>
   <zorder>pushButtonTan</zorder>
   <zorder>checkBox</zorder>
   <zorder>label</zorder>
   <zorder>pushButtonClearAll</zorder>
   <zorder>pushButtonClear</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>derivationDegreeLabel</zorder>
   <zorder>derivationDegreeSpinBox</zorder>
   <zorder>pushButtonLn</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
